packages/rpc/src/methods/stacks/_stacks-helpers.ts

```diff
@@ -5,93 +5,22 @@ export const stacksTransactionDetailsSchema = z.object({
   transaction: z.string(),
 });
 
-// Clarity values
-export const cvIntSchema = z.object({
-  type: z.literal('int'),
-  value: z.string(), // `bigint` compatible
+export const baseStacksTransactionConfigSchema = z.object({
```

Comment:
@kyranjamie I think this should be exported from the package so available. I am having to copy it over rn. I can do, but just flagging.